pages/Login/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { RouteComponentProps } from 'react-router-native';
+import { View, StyleSheet, Image, ViewStyle, TextStyle, TouchableOpacity} from 'react-native';
+import { AppTheme, AppConstants } from '../../config/DefaultConfig';
+import useTheme from "../../hooks/useTheme";
+import ThemedText from '../../components/UI/ThemedText';
+import useConstants from '../../hooks/useConstants';
+import PlaceholderField from '../../components/Chat/PlaceholderField';
+import MaterialIcon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+const Login: React.FunctionComponent<RouteComponentProps> = ({
+    history
+}: RouteComponentProps) => {
+      
+    const goToChatList = () => {
+        history.push('/chatlist')
+    } 
+    const constants: AppConstants = useConstants();
+    const theme: AppTheme = useTheme();
+
+    return (
+        <>
+        <View style={style.mainContainer}>
+            <View style={style.container}>
+                <Image
+                    source={{ uri: constants.appLogo }}
+                    style={{width: 100, height: 100, borderRadius: 50}}
+                />
+            </View>
+            <View style={[style.contentContainer ,{paddingTop: 10, paddingBottom: 30}]}>
+                <ThemedText styleKey="textColor" style={style.nameStyle}>{constants.appName}</ThemedText>
+            </View>
+            <PlaceholderField
+                firstPlaceholder={constants.usernamePlacerHolder}
+                secondPlaceholder={constants.passwordPlacerHolder}
```

Comment:
Again the same problem will be able to resue this in signup? even if we will be able to what about if the signup page doesn't take username?

Reply:
Modified the Code